test/config-next/akamai-purger.json

```diff
@@ -1,7 +1,6 @@
 {
   "akamaiPurger": {
     "debugAddr": ":9666",
-    "purgeInterval": "1ms",
```

Comment:
Shouldn't config-next have a `throughput` stanza now?

Reply:
I actually left it out so we could rely on the optimized defaults. But now that you mention it, I should probably just include the defaults in the config.